gnosis/safe-react
# [L2] Use MultiSendCallOnly instead of MultiSend contract

## What it solves
Resolves #2339 

## How this PR fixes it
Using latest instance for Multisend contract and specially selecting the CallOnly version.

## How to test it
Check that when we execute a Multisend transaction the CallOnly instance is used
CallOnly instance smart contract is deployed at: `0x40A2aCCbd92BCA938b02010E17A5b8929b49130D`

- Test that sending multiple transactions in Transaction Builder is using the instance mentioned above
- Test that upgrading a safe uses the instance mentioned above
- Test that enabling spending limit module and setting a new spending limit uses the multisend instance mentioned above
